deploy/olm-catalog/tektoncd-pipeline-operator/0.3.1/tektoncd-pipeline-operator.v0.3.1.clusterserviceversion.yaml

```diff
@@ -3,21 +3,42 @@ kind: ClusterServiceVersion
 metadata:
   annotations:
     capabilities: Basic Install
+    description: An operator to install TektonCD Pipeline
+    containerImage: quay.io/openshift-pipeline/tektoncd-pipeline-operator:v0.3.1-3
```

Comment:
nitpick: @nikhil-thomas do we want to call the downstream operator "tektoncd-pipeline"-operator ?
@siamaksade WDYT?

Reply:
the display name is 'TektonCD Pipeline Operator'
line 68

Reply:
@sthaha  no. downstream name is "openshift-pipelines-operator"
@nikhil-thomas I sent you a PR to update some of the names and descriptions. Didn't it get merged here? https://github.com/nikhil-thomas/community-operators/pull/1

Reply:
@siamaksade Thank you for the PR. :+1: 

 I shall mergeit in the community operator repo. Then add those changes in this pipeline-repository as well

Reply:
operator-image: quay.io/openshift-pipeline/openshift-pipelines-operator:v0.3.1-1